client/my-sites/plan-features/style.scss

```diff
@@ -386,6 +386,19 @@ $plan-features-sidebar-width: 272px;
 	text-align: center;
 }
 
+.ribbon__wrapper.plan-features__header-ribbon {
```

Comment:
I agree, because it is!
https://github.com/Automattic/wp-calypso/pull/6793/files#diff-d17a13e5e33434c616dceb017c29b76bR48
:)